src/main/java/com/icthh/xm/tmf/ms/activation/service/RetryService.java

```diff
@@ -44,6 +45,7 @@ public void postConstruct() {
     public void rescheduleAllEvents() {
         sagaEventRepository.findByStatus(ON_RETRY).forEach(this::doResend);
         sagaEventRepository.findByStatus(WAIT_DEPENDS_TASK).forEach(this::doResend);
+        sagaEventRepository.findByStatus(TASK_WAIT_CONDITION).forEach(this::doResend);
```

Comment:
suggest rename `TASK_WAIT_CONDITION` --> `WAIT_TASK_CONDITION`

Reply:
or maybe even to `WAIT_CONDITION_TASK`?

Reply:
done